tests/sql_parse_tests.py

```diff
@@ -16,90 +16,96 @@
 # under the License.
 import unittest
 
-from superset import sql_parse
+from superset.sql_parse import ParsedQuery, Table
 
 
 class SupersetTestCase(unittest.TestCase):
     def extract_tables(self, query):
-        sq = sql_parse.ParsedQuery(query)
-        return sq.tables
+        return ParsedQuery(query).tables
+
+    def test_table(self):
+        self.assertEqual(str(Table("tbname")), "tbname")
+        self.assertEqual(str(Table("tbname", "schemaname")), "schemaname.tbname")
+        self.assertEqual(
+            str(Table("tbname", "schemaname", "catalogname")),
+            "catalogname.schemaname.tbname",
+        )
```

Comment:
If we add escaping, I would also test for something along the lines of `Table("tb.name", "schema/name", "catalog\name")`